Src/PerformanceCollector/Shared/Implementation/PerformanceCollectorEventSource.cs

```diff
@@ -72,7 +72,7 @@ public void CounterParsingFailedEvent(string e, string counter, string applicati
 
         [Event(8, Keywords = Keywords.UserActionable, Level = EventLevel.Error,
             Message = @"Error collecting {0} of the configured performance counters. Please check the configuration.
-{2}")]
+{1}")]
```

Comment:
That's what it does, I accidentally broke it with the quick fix which is why we thought we needed to still add that info.